src/pages/Tasks/Tasks.js

```diff
@@ -1,31 +1,102 @@
-import React from "react";
+import React, { useState } from "react";
+import { v4 as uuidv4 } from "uuid";
 import "./task.css";
-import tasksDB from "../../tasksDB";
+import { Link } from "react-router-dom";
 
 const Tasks = () => {
+	const [todos, setTodos] = useState("");
+	const [todoList, setTodoList] = useState([]);
+	const [isEditing, setIsEditing] = useState(true);
+	const [updatedTodo, setUpdatedTodo] = useState(null);
+	const [isCompleted, setIsCompleted] = useState(true);
+
+	const changeHandler = (e) => setTodos(e.target.value);
+
+	const submitHandler = (e) => {
+		e.preventDefault();
+		if (!todos) {
```

Comment:
You dont need to write this check since you are using form and have put the `required` attribute